pkg/project/project_test.go

```diff
@@ -36,6 +34,10 @@ func TestDelete(t *testing.T) {
 
 			// Fake the client with the appropriate arguments
 			client, fakeClientSet := occlient.FakeNew()
+			occlient.SetCurrentProject = func(project string, c *occlient.Client) error {
+				t.Log("Invoked mock SetCurrentProject\n")
```

Comment:
Might be better to have this message when it is not invoked as a check.

Reply:
Yeah that was a debug print I left accidentally in the PR